bin/options.js

```diff
@@ -3,23 +3,58 @@
 /**
  * Dependencies.
  */
-
 const fs = require('fs');
+const path = require('path');
+const readPkgUp = require('read-pkg-up');
 
 /**
  * Export `getOptions`.
+ * Export `getTestDirectory`.
  */
+module.exports = {
+  getOptions: getOptions,
+  getTestDirectory: getTestDirectory
+};
 
-module.exports = getOptions;
+/**
+ * Default test directory.
+ *
+ * @constant
+ * @type {string}
+ * @default
+ */
+const DEFAULT_TEST_DIRECTORY = 'test';
 
 /**
- * Default pathname for run-control file.
+ * Default filename of run-control file.
  *
  * @constant
  * @type {string}
  * @default
  */
-const defaultPathname = 'test/mocha.opts';
+const DEFAULT_OPTS_FILENAME = 'mocha.opts';
+
+/**
+ * Reads directories.test of package.json.
+ *
+ * @public
```

Comment:
I think we should make a rule that every new method should have tag `@private` unless there's a good reason to make it `@public`, it's been discussed.

heck... it'd be nice to have an ESLint rule to enforce this (require `@public` or `@private` in docstrings).. maybe #3583 will help there.